From 26f1af530a7b8f23f93a345028a6f6f5dc28cb9b Mon Sep 17 00:00:00 2001
From: Nitin Garg <nitin.garg@freescale.com>
Date: Wed, 17 Oct 2012 22:37:56 -0500
Subject: [PATCH 02/10] ENGR00230334: Fix the u-boot distclean rule

The u-boot distclean need the CROSS_COMPILE prefix
otherwise it assumes arm-linux- and does not find the
compiler.

Change-Id: If12d94f0b91c4f73bc5dca6f5fc14223f4f90320
Signed-off-by: Nitin Garg <nitin.garg@freescale.com>
---
 core/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/Makefile b/core/Makefile
index 50008fd..fbfd6ae 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -758,7 +758,7 @@ $(TARGET_BOOTLOADER_IMAGE):
 	@for ubootplat in $(TARGET_BOOTLOADER_CONFIG); do \
 		UBOOT_PLATFORM=`echo $$ubootplat | cut -d':' -f1`; \
 		UBOOT_CONFIG=`echo $$ubootplat | cut -d':' -f2`; \
-		$(MAKE) -C bootable/bootloader/uboot-imx/ distclean; \
+		$(MAKE) -C bootable/bootloader/uboot-imx/ distclean $(BOOTLOADER_ENV); \
 		$(MAKE) -C bootable/bootloader/uboot-imx/ $$UBOOT_CONFIG $(BOOTLOADER_ENV); \
 		$(MAKE) -C bootable/bootloader/uboot-imx/ $(BOOTLOADER_ENV); \
 		install -D bootable/bootloader/uboot-imx/u-boot.bin $(PRODUCT_OUT)/u-boot-$$UBOOT_PLATFORM.bin; \
-- 
1.8.0

